logstash-core/lib/logstash/lir_serializer.rb

```diff
@@ -0,0 +1,128 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
This header no longer rings true if we are placing this under the Apache 2.0.license.